### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by wmora**
* Enabled anchor links with [anchorate](https://www.npmjs.com/package/anchorate)
* Fixed broken links from old docs 😭 
* Fixed broken image
* Reintroduced text from https://github.com/smooch/smooch-docs/pull/191

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -16,3 +17,7 @@ export const onRouteUpdate = ({pathname}) => {
         }
     }
 };
+
+export const onRouteChange = () => {
+  anchorate();
```
Comment (lemieux) [inline gatsby-browser.js]: uh I think I coded that logic at some point and removed it for some reason.. oh well

Comment (lemieux) [inline gatsby-browser.js]: actually I didn't delete it, it's right here https://github.com/smooch/smooch-docs/blob/integration/pages/_template.jsx#L13.

I guess it's not reliable after all..?

Comment (wmora) [inline gatsby-browser.js]: hmm I guess not, it's not working in production. I'll remove that logic

File: pages/_template.jsx [hunk 2]
```diff
@@ -10,20 +10,6 @@ export default class extends Component {
         children: PropTypes.any
     };
 
-    scrollToHash = () => {
```
Comment (lemieux) [inline pages/_template.jsx]: don't forget to remove where it's called too!

Comment (wmora) [inline pages/_template.jsx]: 🤦‍♂️

**Pull request merged**